charcoal/Snakefile

```diff
@@ -92,7 +92,7 @@ if provided_lineages_file:
                 print(f'** ({genome_filename} not in {genome_list_file})')
                 if strict_mode:
                     sys.exit(-1)
-            if len(row[1:]) <= 1:
+            if len(row[1:]) < 1:
```

Comment:
this would be more pythonic and less confusing - what do you think?
```suggestion
            if not row[1:]:
```

Reply:
sounds good!